src/scheduler.rs

```diff
@@ -166,75 +166,90 @@ async fn inbox_loop(ctx: Context, started: Sender<()>, inbox_handlers: ImapConne
         .await;
 }
 
-async fn fetch_idle(ctx: &Context, connection: &mut Imap, folder_config: Config) -> InterruptInfo {
```

Comment:
I'd avoid such renamings, ctx is the common abbreviation and the PR is quite big even w/o them